azavea/fb-gender-survey-dashboard
# Download survey data and config directly from HDX

## Overview

The official data sources have been published and can be consumed
directly from the HDX data page.

The new published datasets were processed with our script and as a
result, the country data and config files were modified (regions were
unchanged). It appears one country had whitespace trimmed and was
subsequently re-ordered. Those updates are included here.

Connects #40 

## Testing Instructions

 * On the VM, run `./scripts/dataproc` and see that it runs successfully, and new files are written to `src/dataproc/output`
 * Based on the readme instructions, update the config and data files in the app directory. Run `git status` and ensure that there are no changes (they are added in this PR)
